### ISSUE TITLE
`%hint` not listed as a pragma

### ISSUE THREAD

**Issue by CodingCellist** [idris-lang/Idris2#2766]
(Spotted thanks to mikkom on discord)

`%hint` doesn't seem to show up in the list of pragmas returned by `idris2 --help pragma`. It could be that it isn't considered one, but given that it has the same shape as the other pragmas, I'd expect it to be included

# Steps to Reproduce

Run

```
$ idris2 --help pragma | grep hint
```

# Expected Behavior

```
$ idris2 --help pragma | grep hint
%hint
$
```

# Observed Behavior

```
$ idris2 --help pragma | grep hint
$
```

**Comment by zopa** [comment]
This seems fixed in 0.6.0: 

    ➜ idris2 --help pragma | grep '%hide'   
    + %hide nm

Possibly by 9797a79b53b8a4c7d0f1f8d23c1a2ca672b29583?

**Comment by mattpolzin** [comment]
hide /= hint 🙂
